src/citrine/informatics/design_spaces/formulation_design_space.py

```diff
@@ -63,15 +63,15 @@ def __init__(self, *,
                  description: str,
                  formulation_descriptor: FormulationDescriptor,
                  ingredients: Set[str],
-                 constraints: Set[Constraint],
+                 constraints: List[Constraint],
                  labels: Optional[Mapping[str, Set[str]]] = None,
                  resolution: float = 0.0001,
                  session: Session = Session()):
         self.name: str = name
         self.description: str = description
         self.formulation_descriptor: FormulationDescriptor = formulation_descriptor
         self.ingredients: Set[str] = ingredients
-        self.constraints: Set[Constraint] = constraints
+        self.constraints: List[Constraint] = constraints
```

Comment:
This should still be a Set

Reply:
Why, out of curiosity?

Reply:
Order doesn't matter. But also, it was a Set before, so we're not changing anything. I changed it to List as part of some messing around.